### PR TITLE
Feature/1906 - Rework tables so all have same styling and move results to bottom

### PR DESCRIPTION
**Pull request by sasha-dresden**
Issue: #1906 
API: [PR889](https://github.com/fecgov/fecfile-web-api/pull/889)

### CODE REVIEW

File: front-end/src/app/shared/components/table/table.component.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        <td *ngIf="hasCheckbox">
+          <p-tableCheckbox [value]="item" [ariaLabel]="checkboxLabel!(item)"></p-tableCheckbox>
+        </td>
+        <ng-container *ngTemplateOutlet="body; context: { $implicit: item }"></ng-container>
+      </tr>
+    </ng-template>
+    <ng-template pTemplate="summary">
+      <div class="d-flex justify-content-between">
+        <div class="responsive-container">
+          <span class="paginator-text">Results per table:</span>
```
Comment (mjtravers) [inline front-end/src/app/shared/components/table/table.component.html]: Should this be "Results per page:"?

File: front-end/src/app/shared/components/table/table.component.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+          </p-paginator>
+        </div>
+      </div>
+    </ng-template>
+    <ng-template pTemplate="sorticon" let-sortOrder>
+      <app-table-sort-icon [sortOrder]="sortOrder"></app-table-sort-icon>
+    </ng-template>
+    <ng-template pTemplate="paginatorleft">
+      <div class="responsive-container">
+        <span>Results per table:</span>
```
Comment (mjtravers) [inline front-end/src/app/shared/components/table/table.component.html:100]: Should this be "Results per page:"?

**Review by mjtravers — CHANGES_REQUESTED** [review]
A small label change requested. Also, what is the default number of rows to show on a table, 5 or 10?

File: front-end/src/app/shared/components/table/table.component.html (hunk 1 above)
Comment (sasha-dresden) [reply front-end/src/app/shared/components/table/table.component.html]: Ah I didn't catch that. The old way was "table" and there wasn't a specific point to change it to "page" however all the wireframes had "page". Changed it to page.

Comment (sasha-dresden) [reply front-end/src/app/shared/components/table/table.component.html]: As for the default number of rows, there isn't one. Some pages had their default as 10, others as 5.

**Review by mjtravers — APPROVED** [review]

**Pull request merged**